sdk/eventhub/event-hubs/README.md

```diff
@@ -189,10 +172,14 @@ const partitionIds = await client.getPartitionIds();
 const subscription = client.subscribe(
   EventHubClient.defaultConsumerGroupName,
   (receivedEvents, context) => { },
-  [ partitionIds[0] ], {
+  partitionIds[0], {
     defaultEventPosition: EventPosition.latest()
   }
 );
+
+// events will be fired off as messages are received
+
+subscription.close();
```

Comment:
Isnt this async? I yes, then we should add an await

Reply:
On second thoughts... why do we need this code snippet here when we have the `Subscribe with an event handler` a few lines below?